react-main/src/redux/store.js

```diff
@@ -0,0 +1,9 @@
+import { combineReducers, createStore } from 'redux';
+
+import { historyReducer } from './history/reducer';
+
+const mainReducer = combineReducers({ historyReducer });
+
+const mainStore = initialState => createStore(mainReducer, initialState);
```

Comment:
Since initialState isn't necessary, i would define mainStore as `const mainStore = createStore(mainReducer)`. Then, i would import it and use it directly in the main index